client/jetpack-cloud/sections/manage/pricing/pricing-license-selector/style.scss

```diff
@@ -0,0 +1,55 @@
+.pricing-license-selector-wrapper {
```

Comment:
A tiny suggestion:  `.pricing-license-selector`, `.pricing-license-option-outer`, `.pricing-license-option`, `.pricing-license-option.is-selected` can all sit within `.pricing-license-selector-wrapper  `.

For example: 
```
.pricing-license-selector-wrapper {
	display: flex;
	width: 100%;
	height: 72px;
	background: rgba(255, 255, 255, 0.84);
	position: sticky;
	top: 0;
	z-index: 1;
	backdrop-filter: blur(4px);

	.pricing-license-selector {
...

	}
}
```

Reply:
Great suggestion, thanks! Addressed in ab4c593.